project_one/index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title>Document</title>
+    <link rel="stylesheet" href="index.css">
+</head>
+<body>
+
+    <header>
+        <div class ="pageheader"><strong>MuralFind</strong></div>
+        <nav>
+        <ul>
+            <li><a href = "index.html">Home</a></li>
+            <li><a href = "About.html">About</a></li>
+            <li><a href = "Map.html">Map</a></li>
+            <li><a href = "FAQ.html">FAQ</a></li>
+        </ul>
+
+    </nav>
+
+        
+    </header>  
+    <div class = "banner">
+            <img src="https://images.unsplash.com/photo-1561898329-9ad7d76fbb2a?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1050&q=80" alt="Italian Trulli">
+            <img src="https://images.unsplash.com/photo-1551363786-c8bc9e05170c?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1050&q=80">
+            <img src ="https://www.learnodo-newtonic.com/wp-content/uploads/2017/09/Balloon-Girl-2002-Banksy.jpg">
+    </div>
+
+<div class = "call-to-action">
+    <h4>Find Art Murals Closest to you</h4>
+    <p>Muralfind was created to help users find street art and raise awareness</p>
+    
+    <h5><a id = "grading-element-one" onclick="addtoDOM()">Why murals?</a></h5>
```

Comment:
correctly adds an element to DOM, but i could not find it with `document.querySelector("#grading-element-one-consequence")`